pkg/core/extension_store.go

```diff
@@ -523,6 +550,8 @@ func (store *ExtensionStore) UpdateCredentials(ctx context.Context, credentials
 	if err != nil {
 		return err
 	}
+	credentials.Created = credRecord.Created
```

Comment:
Again, why even touch Created on Update operation?

Reply:
The next operation overwrites everything in the in-memory database with the content supplied as input.